summingbird-storm/src/main/scala/com/twitter/summingbird/storm/FlatMapBoltProvider.scala

```diff
@@ -162,4 +156,15 @@ case class FlatMapBoltProvider(storm: Storm, jobID: JobId, stormDag: Dag[Storm],
       case None => getIntermediateFMBolt[Any, Any].asInstanceOf[BaseBolt[Any, Any]]
     }
   }
+
+  private def inputEdges[Input]: Map[String, Edge[Input]] = {
```

Comment:
seems like this should be passed in on construction from the nodes it depends on. Is that something we will get to? A win, seems to me, would be we share the same `Edge` instances between the output at one level and then input at the next, and it could reduce the chance for error there.